community/sacc/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Sascha Paunovic <azarus@posteo.net>
+# Maintainer: Sascha Paunovic <azarus@posteo.net>
+pkgname=sacc
+pkgver=1.00
+pkgrel=0
+pkgdesc="saccomys, simple console gopher client"
+url="gopher://bitreich.org/1/scm/sacc"
+arch="all"
+license="ISC"
+depends="ncurses"
+makedepends="ncurses-dev"
+subpackages="$pkgname-doc"
+source="ftp://bitreich.org/releases/$pkgname/$pkgname-v$pkgver.tgz"
+builddir="$srcdir/$pkgname-v$pkgver"
+
+build() {
+	make -C "$builddir"
+}
+
+package() {
+	cd "$builddir"
+	install -Dm644 LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
```

Comment:
ISC is standard OSI-approved license, so it’s not needed to install the LICENSE file.